pandas/tests/series/test_api.py

```diff
@@ -167,3 +167,15 @@ def test_attrs(self):
         s.attrs["version"] = 1
         result = s + 1
         assert result.attrs == {"version": 1}
+
+    def test_series_with_object_dtype(self):
```

Comment:
these should go in pandas/tests/series/indexing/test_setitem.py

name the test with _empty in its name